src/components/ItemFilters/FiltersContainer.tsx

```diff
@@ -78,62 +108,83 @@ const ItemFilterContainer = ({ itemAggs }: ItemFilterContainerProps) => {
           display: "flex",
           flexDirection: isLargerThanMedium ? "row" : "column",
         }}
+        mb="m"
       >
-        <Card className={filterGroupClassName} ref={ref}>
-          <CardHeading level="h3" size="body2">
-            Filter by
-          </CardHeading>
-          <CardContent>
-            <Box
-              sx={{
-                display: "flex",
-                flexDirection: isLargerThanLarge ? "row" : "column",
-              }}
-              gap="nypl-s"
-            >
-              {filterData.map((field: ItemFilterData) => (
-                <ItemFilter
-                  isOpen={whichFilterIsOpen === field.field}
-                  setWhichFilterIsOpen={setWhichFilterIsOpen}
-                  key={field.field}
-                  itemFilterData={field}
-                  appliedFilters={appliedFilters}
-                  submitFilters={submitFilters}
-                />
-              ))}
-            </Box>
-          </CardContent>
-        </Card>
-        <Card className={filterGroupClassName}>
-          <CardHeading
-            level="h3"
-            size="body2"
-            data-testid="filter-text"
-            isBold={true}
+        <Box className={filterGroupClassName} ref={ref}>
+          <Label
+            id="filters-label"
+            htmlFor="item-filters"
+            fontWeight="bold"
+            data-testid="filters-label"
+          >
+            Filter By
+          </Label>
+          <Box
+            sx={{
+              display: "flex",
+              flexDirection: isLargerThanLarge ? "row" : "column",
+            }}
+            gap="nypl-s"
+            id="item-filters"
+          >
+            {filterData.map((field: ItemFilterData) => (
+              <ItemFilter
+                isOpen={whichFilterIsOpen === field.field}
+                setWhichFilterIsOpen={setWhichFilterIsOpen}
+                key={field.field}
+                itemFilterData={field}
+                appliedFilters={appliedFilters}
+                submitFilters={submitFilters}
+              />
+            ))}
+          </Box>
+        </Box>
+        <Box className={filterGroupClassName} minWidth={440}>
+          <Label
+            id="year-filter-label"
+            htmlFor="year-filter"
```

Comment:
Not a fan of this but I know why you're doing this. Not your fault but something I need to chat with Marty about regarding showing labels in the `SearchBar` component.